build/tutorials-contest/liquid-avax/Backend/backend.js

```diff
@@ -0,0 +1,272 @@
+// This script will be used to do the cross-chain transfer of the owner wallet from C to P chain and stake them
+// to the user-chosen node.
+
```

Comment:
Generally we prefer typescript instead of javascript. When writing crypto code you should always use types. 

1. Port to typescript
2. Add types for every variable declaration and return value
3. Remove semicolons
4. Make the code self documenting by using `const` for every variable which will remain a constant value and `let` only when a variable will change it's value later in the script